components/accept-financial-contributions/AcceptContributionsMyself.js

```diff
@@ -94,9 +107,19 @@ class AcceptContributionsMyself extends React.Component {
       bankInformation: '',
     };
 
-    const submit = values => {
-      const { bankInformation } = values;
-      this.submitBankAccountInformation(bankInformation);
+    const submit = async values => {
+      try {
+        const { bankInformation } = values;
+        await this.submitBankAccountInformation(bankInformation);
+        await this.addHost(collective, LoggedInUser.collective);
```

Comment:
Maybe it would be better to `refetchLoggedInUser` only after this line so that we don't do it 2 times.